SPECS/python-tensorboard/python-tensorboard.spec

```diff
@@ -0,0 +1,110 @@
+%global pypi_name tensorboard
+%global _description %{expand:
+TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs}
+%define _enable_debug_package 0
+%global debug_package %{nil}
+
+Summary:        TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs
+Name:           python-%{pypi_name}
+Version:        2.11.0
```

Comment:
Just double checking that this version number is correct.  It's the same version as the next spec file.

Reply:
TensorFlow, Tensorboard, Keras, and most of the dep of Tensorflow made by the TensorFlow team have the same version. For example in this case 2.11.0